elasticsearch/values.yaml

```diff
@@ -129,8 +129,20 @@ updateStrategy: RollingUpdate
 # of your pods to be unavailable during maintenance
 maxUnavailable: 1
 
- # GroupID for the elasticsearch user. The official elastic docker images always have the id of 1000
-fsGroup: 1000
+podSecurityContext:
+  fsGroup: 1000
+
+# The following value is deprecated,
+# please use the above podSecurityContext.fsGroup instead
+fsGroup: {}
```

Comment:
This should be an empty value and not an empty hash I think.

Reply:
Set to `null`!